backend/tests/v5/e2e/routes/teamspaces/projects/containers/revisions.test.js

```diff
@@ -88,14 +92,15 @@ const setupData = async () => {
 		ServiceHelper.db.createUser(nobody),
 		ServiceHelper.db.createProject(teamspace, project.id, project.name, models.map(({ _id }) => _id)),
 		...revisions.map((revision) => ServiceHelper.db.createRevision(teamspace, modelWithRev._id, revision)),
+		ServiceHelper.db.createRevisionRef(teamspace, modelWithRev._id, `${validRefTypeRevision._id}P2006823-BDP-XX-A-Stanmore-Envelope_optimized_ifc`),
```

Comment:
let createRevision do this if there's a file ref, just pass data with it to store